utils/scripts/helpers/repo-helper-ubuntu.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -uex
+
+# This script is used by dockerfiles to optionally use
+# a local repository instead of a distro provided repository.
+# It will also optionally allow running a /tmp/install script
+# for custom packages if present.
+
+: "${REPO_FILE_URL:=}"
+: "${HTTPS_PROXY:=}"
+: "${DAOS_LAB_CA_FILE_UR:=}"
+: "${REPOSITORY_NAME:=artifactory}"
+
+disable_repos () {
+    if [ -e /etc/apt/sources.list.d/ubuntu.sources ];then
+        mv /etc/apt/sources.list.d/ubuntu.sources \
+           etc/apt/sources.list.d/ubuntu.sources.disabled
+    elif [ -e /etc/apt/sources.list ];then
+        mv /etc/apt/sources.list \
+           etc/apt/sources.list.disabled
+    fi
+}
+
+# Use local repo server if present
+install_curl() {
+
+    if command -v curl; then
+        echo "found curl!"
+        return
+    else
+        apt-get update
+        apt-get install curl ca-certificates gpg gpg-agent \
+                software-properties-common
+    fi
+
+    if command -v wget; then
+        echo "found wget!"
+        return
+    fi
```

Comment:
Is this necessary here? Maybe copy-paste leftover?

Reply:
Needed for compatibility for upcoming PRs for the packaging repos.
We can still build some Ubuntu dependencies, and we should keep that code in step.

One Ubuntu Docker image can actually be used for multiple Ubuntu pbuilder builds, but that will take a larger rewrite.